src/main/daml/Daml/Finance/Interface/Account/Account.daml

```diff
@@ -84,7 +82,7 @@ interface Account where
     with
       quantity : Quantity InstrumentKey Decimal
         -- ^ The target `Instrument` and corresponding amount.
-    controller (view this).custodian, (view this).owner
+    controller (view this).custodian, (view this).controllers.approvers
```

Comment:
I need a bit more time to understand why this asymmetry we are introducing (with respect to owner and custodian) is necessary...

Reply:
I think it comes from the fact that, on transfers, we force the custodian to be the same

Reply:
Just to highlight why I think this is important:
- in the asset model documentation for an OTC, we state that it doesn't really matter who is who for `instrument` and `account`: it only matters in holdings for the direction of the payoff
- however, looking at the implementation of account, it appears that the `custodian` has a special role

Clearly, the symmetry can be restored by setting `instructors = approvers = owner`

Reply:
reason is here:
https://github.com/digital-asset/daml-finance/pull/424#discussion_r1034656266

Reply:
Yup, and the asymmetry was already there before (in the `debitAndCredit` choice)

Reply:
yes, exactly. the `DebitAndCredit` choice opened up the`Credit` and `Debit` choices (which had `custodian` and `owner` as controller). 

With the current change, the `custodian` can now call Credit` and `Debit` directly (for an `owner`). But before that would be possible as well over a detour: 
1. for a `Credit`: the `custodian` first spawns a holding for **itself** and then call `DebitAndCredit` 
2. for a `Debit`: the `custodian` exercises `DebitAndCredit` using an account for **itself**

I think it is easier to understand now though, and the `Transferable` interface is not only there for connivence (but actually captures the transfer logic except for the controllers).